Lib/test/test_structmembers.py

```diff
@@ -106,6 +106,14 @@ def test_bad_assignments(self):
         for nonint in None, 3.2j, "full of eels", {}, []:
             for attr in integer_attributes:
                 self.assertRaises(TypeError, setattr, ts, attr, nonint)
+        #for nonint in 3.2j,:
```

Comment:
What's the point of this commented-out code?

Reply:
No point. This was left after testing an intermediate version. Good catch!